releases/release-1.22/release-notes/maps/pr-90368-map.yaml

```
@@ -0,0 +1,5 @@
+pr: 90368
+releasenote:
+  text: "kube-controller-manger: the `--horizontal-pod-autoscaler-use-rest-clients`
```

🤔 i think that's a typo

```suggestion
  text: "kube-controller-manager: the `--horizontal-pod-autoscaler-use-rest-clients`
```

---

Are we sure [manger](https://en.m.wikipedia.org/wiki/Manger#:~:text=A%20manger%20or%20trough%20is,found%20at%20stables%20and%20farmhouses.) Is not a better # name 🤠🤠🤠, good catch!

---

Thanks for the catch! Updated.